samples/bluetooth/peripheral/prj.conf

```diff
@@ -4,6 +4,6 @@ CONFIG_BLUETOOTH_SMP=y
 CONFIG_BLUETOOTH_SIGNING=y
 CONFIG_BLUETOOTH_PERIPHERAL=y
 CONFIG_BLUETOOTH_ATT_PREPARE_COUNT=2
-CONFIG_BLUETOOTH_GATT_DYNAMIC_DB=y
 CONFIG_BLUETOOTH_PRIVACY=y
 CONFIG_BLUETOOTH_DEVICE_NAME="Test peripheral"
+CONFIG_BLUETOOTH_DEVICE_APPEARANCE=0x0341
```

Comment:
This doesn't actually work. You've declared the Kconfig as 'int' instead of 'hex' and so the above results in the following in .config:

CONFIG_BLUETOOTH_DEVICE_APPEARANCE=0